sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -189,7 +189,10 @@ export class ApplicationStudentsController extends BaseController {
       );
       if (notValidDates) {
         throw new UnprocessableEntityException(
-          `${INVALID_STUDY_DATES} ${notValidDates}`,
+          new ApiProcessError(
+            `${INVALID_STUDY_DATES} ${notValidDates}`,
```

Comment:
INVALID_STUDY_DATES is the error code. 
It has to be used like this

`throw new UnprocessableEntityException(
          new ApiProcessError(
            notValidDates,
            INVALID_STUDY_DATES,
          ),
        );`